install/prepare_site_init.md

```diff
@@ -1,3 +1,4 @@
+<!-- markdownlint-disable MD013 -->
```

Comment:
These are very unfriendly to terminals.

Reply:
Since we include image assets throughout the docs, I would argue that the docs are not designed for consumption within a terminal.

Reply:
What am I missing? Vim and less both wrap text and the bulk of our text isn't code, it's copy in paragraphs. We also include image assets in the docs without alt text. If we wanted to target the terminal, wouldn't we be post-processing the markdown and rendering it into txt documents?

Don't we want to eliminate extra newlines so that we can format and stylize the text properly in the PDF and HTML versions of the docs without the syntax driving the formatting?

Reply:
We moved this conversation to offline, and the consensus was to let the docs-team discuss disabling this check globally for docs-csm. We don't see the newlines affecting the hugo rendered docs, but in general we aren't aware of non-wrapped usage of the raw doc files (where the line length is helpful).